src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/CannotConfirm.cshtml.cs

```diff
@@ -0,0 +1,11 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace SFA.DAS.ApprenticeCommitments.Web.Pages.Apprenticeships
+{
+    public class CannotConfirmApprenticeshipModel : PageModel
+    {
+        [BindProperty(SupportsGet = true)]
+        public string? ApprenticeshipId { get; set; }
```

Comment:
ApprenticeshipId probably shouldn't be optional.

Reply:
Waiting to rework this when we implement the HashedId